crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -634,11 +634,11 @@ impl CumulocityConverter {
     ) -> Result<Vec<MqttMessage>, ConversionError> {
         // JSON over MQTT messages on c8y/devicecontrol/notifications can contain multiple operations in a single MQTT
         // message, so split them
-        let operation_payloads = collect_c8y_messages(message.payload_str()?);
+        let operation_payloads = message.payload_str()?.lines();
```

Comment:
Okay - assuming that Cumulocity emits a batch of messages using JSON lines. A point I fail to find in Cumulocity docs.

Reply:
Unfortunately I don't think this is publicly documented.